src/index.js

```diff
@@ -11,6 +11,11 @@ injectGlobal`
   * {
     box-sizing: border-box;
   }
+
+  body, html {
+    font-size: 16px;
+    font-family: 'Roboto', sans-serif;
```

Comment:
I think the main typeface is `Lato` as said at https://github.com/bblfsh/artwork/issues/1#issuecomment-309639525